types/mongodb/index.d.ts

```diff
@@ -34,6 +34,7 @@
 //                 Anna Henningsen <https://github.com/addaleax>
 //                 Emmanuel Gautier <https://github.com/emmanuelgautier>
 //                 Wyatt Johnson <https://github.com/wyattjoh>
+//                 Boris Figovsky <https://github.com/borfig>
```

Comment:
Note that this means that you want to be notified of future changes to these types so that you can provide feedback (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#definition-owners).

Reply:
I am aware of this. I have done the same at @types/mongoose

Reply:
Thanks for confirming!